databuilder/extractor/dashboard/mode_dashboard_extractor.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+from requests.auth import HTTPBasicAuth
+from typing import Any  # noqa: F401
+
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.restapi.rest_api_extractor import RestAPIExtractor, REST_API_QUERY, MODEL_CLASS
+from databuilder.rest_api.base_rest_api_query import RestApiQuerySeed
+from databuilder.rest_api.rest_api_query import RestApiQuery
+
+# CONFIG KEYS
+ORGANIZATION = 'organization'
+MODE_ACCESS_TOKEN = 'mode_user_token'
+MODE_PASSWORD_TOKEN = 'mode_password_token'
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ModeDashboardExtractor(Extractor):
+    """
+    A Extractor that extracts core metadata on Mode dashboard.
```

Comment:
not sure if everyone is aware of what is `Mode`, may worth putting https://app.mode.com/ in the comment?

Reply:
Good point. Will do.